src/OctoshiftCLI.Tests/gei/Commands/DownloadLogsCommandTests.cs

```diff
@@ -155,6 +157,62 @@ public async Task Calls_Download_With_Expected_Migration_Log_File()
             mockHttpDownloadService.Verify(m => m.Download(It.IsAny<string>(), migrationLogFile));
         }
 
+        [Fact]
+        public async Task Waits_For_URL_To_Populate_When_Wait_Requested()
```

Comment:
nit: Use `Url` instead of `URL`. More than two letter identifiers should be `Pascal` cases see [this](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions)

Reply:
Updated with 5cd1977f6e1d19eff4bffb6c2b3e8a879eb5ab60!